electrum/commands.py

```diff
@@ -528,20 +528,22 @@ def _resolver(self, x, wallet):
         message = util.to_bytes(message)
         return ecc.verify_message_with_address(address, sig, message)
 
-    def _mktx(self, wallet, outputs, *, fee=None, feerate=None, change_addr=None, domain=None,
+    def _mktx(self, wallet, outputs, *, fee=None, feerate=None, change_addr=None, domain_addr=None, domain_coins=None,
               nocheck=False, unsigned=False, rbf=None, password=None, locktime=None):
         if fee is not None and feerate is not None:
             raise Exception("Cannot specify both 'fee' and 'feerate' at the same time!")
         self.nocheck = nocheck
         change_addr = self._resolver(change_addr, wallet)
-        domain = None if domain is None else map(self._resolver, domain)
+        domain_addr = None if domain_addr is None else map(self._resolver, domain_addr)
         final_outputs = []
         for address, amount in outputs:
             address = self._resolver(address, wallet)
             amount = satoshis(amount)
             final_outputs.append(TxOutput(TYPE_ADDRESS, address, amount))
 
-        coins = wallet.get_spendable_coins(domain, self.config)
+        coins = wallet.get_spendable_coins(domain_addr, self.config)
+        if domain_coins is not None:
+            coins = list([coin for coin in coins if (coin['prevout_hash'] + ':' + str(coin['prevout_n']) in domain_coins)])
```

Comment:
The `list()` cast looks redundant. Are you trying to make a copy?
```suggestion
            coins = [coin for coin in coins if (coin['prevout_hash'] + ':' + str(coin['prevout_n']) in domain_coins)]
```

Reply:
Good point.  Not sure why I was casting there.  Removed it.